src/js/converter.js

```diff
@@ -146,7 +146,7 @@ return function(writer) {
 			openingTag += '>';
 			array.push(openingTag);
 			array.push('</'+tag+'>');
-		} else if (node.prop('tagName').toLowerCase() == 'br') {
+		} else if (node.length && node.prop('tagName').toLowerCase() == 'br') {
```

Comment:
If a schema is loaded and the root element of the document doesn't match the schema then node is empty.